src/modules/75-cf/pages/environment-details/EnvironmentDetails.tsx

```diff
@@ -108,6 +108,17 @@ const EnvironmentDetails: React.FC = () => {
               />
             )
           }
+          pagination={
+            <Pagination
+              itemCount={keys?.itemCount ?? 0}
+              pageCount={keys?.pageCount ?? 0}
+              pageIndex={keys?.pageIndex ?? 0}
+              pageSize={CF_DEFAULT_PAGE_SIZE}
+              gotoPage={(index: number) => {
+                setPage(index)
+              }}
+            />
```

Comment:
You'll want to avoid passing this at all if there are no items
```suggestion
            !!keys?.itemCount && (
              <Pagination
                itemCount={keys.itemCount}
                pageCount={keys?.pageCount || 1}
                pageIndex={keys?.pageIndex || 0}
                pageSize={CF_DEFAULT_PAGE_SIZE}
                gotoPage={setPage}
              />
            )
```